src/Fmt_ast.ml

```diff
@@ -1010,6 +1010,12 @@ and fmt_expression c ?(box= true) ?epi ?eol ?parens ?ext
         ( fmt_expression c (sub_exp ~ctx s) $ fmt ".["
         $ fmt_expression c (sub_exp ~ctx i) $ fmt "]@ <- "
         $ fmt_expression c (sub_exp ~ctx e) )
+  | Pexp_apply
+      ( {pexp_desc= Pexp_ident {txt= Lident ":="}; pexp_attributes= []}
+      , [(Nolabel, r); (Nolabel, v)] ) ->
```

Comment:
You probably need to add `hovbox 0` here analogous to the general case below, or else get cases like
```
;; if cccccccccccccccccccc then x :=
       f aaaaaaaaaaaaaaaaaaaa bbbbbbbbbbbbbbbbbbbb
```

Reply:
indeed